ui/src/app/icons/scroll/component.tsx

```diff
@@ -0,0 +1,50 @@
+import {Component, Input} from '@angular/core';
+import {ReactWrapperBase} from 'app/utils';
+
+import {ScrollIconReact} from 'app/icons/scroll-icon/component';
+import * as React from 'react';
+import * as Interactive from 'react-interactive';
+
+const FillColorId = '#2691D0';
+const HoverFillColorId = '#72B9E2';
+const LightFillColorId = '#5FAEE0';
+const LightHoverFillColorId = '#1892E0';
+
+const wrapper = ScrollIconReact;
+export const ScrollReact: React.FunctionComponent<{icon: string, shade: string, style: object}> =
+    ({icon, shade, style}) => {
+      const transform = (icon === 'RightIcon') ? 'scaleX(-1)' : '';
+      let color = FillColorId;
+      let hoverColor = HoverFillColorId;
+      let opacity = '1';
+      let width = '47';
+      let height = '48';
+
+      if (shade === 'Light') {
+        color = LightFillColorId;
+        hoverColor = LightHoverFillColorId;
+        opacity = '0.54';
+        width = '40';
+        height = '41';
+      }
+
+      return  <div>
+        <Interactive as={wrapper}
+          style={{color: color, opacity: opacity, transform: transform, ...style}}
+          hover={{color: hoverColor}} width= {width} height={height}/>
+      </div>;
```

Comment:
Since there are a lot of differences between the two cases, it might be cleaner to express this as a single conditional, and then inline the relevant props/style. That would allow you to avoid using `let` above (which should be avoided unless you really need mutation).

E.g.
```
kind === 'one' ?
  <Foo a='1' b='2' c='3' /> :
  <Foo a='7' b='8' c='9' />
```